src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const personMaried = isMarried
+    ? (`${partnerName} is my ${
+      sex === 'm'
+        ? ('wife')
+        : ('husband')}`)
+    : ('I am not married');
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        { `My name is ${name}`}
+      </h2>
+
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      )}
```

Comment:
nested elements should have indents
```suggestion
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```